code/modules/client/client_procs.dm

```diff
@@ -1240,6 +1224,29 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 
 		screen -= object
 
+/// Handles any "fluff" or supplementary procedures related to an admin logout event. Should not have anything critically related cleaning up an admin's logout.
+/client/proc/handle_admin_logout()
+	adminGreet(logout = TRUE)
+	if(length(GLOB.admins) > 0 || !SSticker.IsRoundInProgress()) // We only want to report this stuff if we are currently playing.
+		return
+
+	var/cheesy_message = pick(list(
```

Comment:
this should pick from a static list.
no reason to be instancing a new list every time